### ISSUE TITLE
Forms9Patch.Label does not support the Xamarin.Forms Previewer

### ISSUE THREAD

**Issue by renzska** [baskren/Forms9Patch#30]
When running the Xamarin.Forms Previewer against the Forms9Patch.Label, it does not display and writes the following to the logs:

> Forms9Patch.Label is not DesignTimeVisible, nor does it implement Component

I'm guessing that other components in Forms9Patch do the same thing as this was just one of my simple test pages.

This article explains that the [DesignTimeVisible(true)] attribute is needed for custom controls to render:

[https://docs.microsoft.com/en-us/xamarin/xamarin-forms/xaml/xaml-previewer/render-custom-controls](url)

I'd like to request adding support to the controls for Xamarin.Forms Previewer.

Thanks.

**Comment by baskren** [comment]
@renzska -  Thanks for pointing me to the doc.  I had asked James Montemagmo via twitter how to do this about a year ago.  I don't blame James for not responding (like many other things in our modern world, I blame twitter).

Anyway, back on topic - I've just pushed this to the BetterDispose branch.  Since it's not clear to me what's happening under the covers (a search in the Xamarin.Forms github repo for DesignTimeVisible came up empty), I'm not 100% sure I got this right everywhere.  I did verify that it is working for Forms9Patch.Label.
